src/libprojectM/projectM.cpp

```diff
@@ -827,7 +826,7 @@ void projectM::populatePresetMenu()
             int presetIndex = getSearchIndex(presetName);
             for (unsigned int i = 0; i < getPlaylistSize(); i++)
             { // loop over all presets
-                if (caseInsensitiveSubstringFind(getPresetName(i), renderer->searchText()) != std::string::npos)
+                if (CaseInsensitiveSubstringFind(getPresetName(i), renderer->searchText()) != std::string::npos)
```

Comment:
Just to note: this already fails with Unicode/UTF-8 preset names (which aren't even displayed correctly), and ideally shouldn't be in the core library at all. Moving it to a helper library and making it multi-byte-aware would be a good idea later on...